docs/make-changes-to-brexit-checker-content.md

```diff
@@ -50,19 +43,8 @@ bundle exec rake brexit_checker:convert_csv_to_yaml:actions[path/to/actions.csv]
 
 ## Adding change notes
 
-When making changes to actions, you may also need to create a change note.
-
-This will involve adding an entry to the [lib/brexit_checker/changenotes.yaml](https://github.com/alphagov/finder-frontend/tree/master/lib/brexit_checker/changenotes.yaml)
-file. See the yaml file for examples.
-
-The text should be provided by the content editor. This content will be
-included in emails to users.
-
-If a change note is created, users will be emailed about the change. Without
-a change note, users will not be emailed.
-
-Don't modify change notes, always add new ones. Change note IDs must be unique.
+Additions or changes to actions may require a change note, which are defined [in a `change_notes.yaml` file](https://github.com/alphagov/finder-frontend/blob/master/lib/brexit_checker/change_notes.yaml). You should check with the person who requested the change, to determine if a new entry is appropriate.
 
-## Send updates to subscribed users
+When a new entry has been deployed to production, you need to run a rake task to send the notification. The notification will be sent to all subscribers who would see this action on their results page.
```

Comment:
```suggestion
When a new change note has been deployed to production, you need to run a rake task to send the notification. The notification will be sent to all subscribers who would see this action on their results page.
```

Reply:
Updated 👍. I was initially trying to avoid the ambiguity between the entry and the optional note :-|.